src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/checkoutConfiguration.js

```diff
@@ -180,9 +190,10 @@ function getGiftCardConfig() {
         },
       });
     },
-    onSubmit(state, component) {
+    onSubmit(state) {
       store.selectedMethod = state.data.paymentMethod.type;
-      store.brand = component?.displayName;
+      store.brand = brandArray?.filter((item, i, ar) => ar.indexOf(item) === i)
```

Comment:
Is there a way to make this lien a little easier to read?

Reply:
this was reverted to the previous implementation.